requirements_all.txt

```diff
@@ -564,7 +564,7 @@ envoy_reader==0.16.1
 ephem==3.7.7.0
 
 # homeassistant.components.epson
-epson-projector==0.1.3
+epson-projector==0.2.3
```

Comment:
For a dependency bump, add a link to the release notes to the description of the PR

Reply:
do you mean link to release notes of the library epson-projector?

Reply:
Yes